packages/css/src/components/card-grid/card-grid.scss

```diff
@@ -1,6 +1,8 @@
+@use "../grid/grid.variables" as grid;
```

Comment:
willen we toch maar wel meteen de namespacing goed zetten? `as grid` -> `as grid-variables`